markdown/org/docs/patterns/charlie/instructions/en.md

```diff
@@ -194,17 +194,28 @@ When you're done, press open the inseam.
 
 ## Step 9:  Close the crossseam
 
+<tip>
+
+Double check that your right and left legs match the pattern markings. The right leg's crotch should extend slightly beyond the left. Refer to the pattern and cut excess on the left leg if necessary.
+
+It can be valuable to mark with chalk or a running stitch the **Center Front** on each leg. This will help you make sure the peices are in the right place duing fly construction.
+
+</tip>
+
 Make sure one leg is turned with the good side out, and the other has the good side in.
 
 Now tuck the leg with the good side out inside the leg that has the good side in.
 This way, they have their good sides against each other.
 
 Align the cross seam, starting at the back waist, pinning both halves together as you make your way
-towards the fly. Stop at the lowest fly notch.
+towards the fly. **Stop at the fly notch** and makes sure to back-stitch.
```

Comment:
```suggestion
towards the fly. **Stop at the fly notch** which is the bottom notch, making sure to back-stitch at the notch.
```

Feels best to indicate which notch this is in relevances to the others.

Reply:
In this case it's actually the top notch, but yes :)